nodes/clusters/nodes-cluster-enabling-features.adoc

```diff
@@ -5,11 +5,10 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-As an administrator, you can turn on features that are in Technology Preview
-status.
+As an administrator, you can use feature gates to enable features that not part of the default set of features.
```

Comment:
s/to enable features that not part of/to enable features that **are** not part of/